starknet-foundry/crates/cast/src/starknet_commands/invoke.rs

```diff
@@ -73,7 +50,7 @@ pub async fn execute_calls(
     account: &SingleOwnerAccount<&JsonRpcClient<HttpTransport>, LocalWallet>,
     calls: Vec<Call>,
     max_fee: Option<FieldElement>,
-) -> Result<FieldElement> {
+) -> Result<Vec<(&'static str, String)>> {
```

Comment:
Won't returning strings everywhere make it harder to reuse this later in deployment scripts?

Reply:
Unfortunately, I don't know :/ But we are returning name and value, so we can make FieldElements from Strings then. If there are problems with those returning values we will think of another solution.

Reply:
Changed to the structs 🔥